python/django/python-hello-world/.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6" project-jdk-type="Python SDK" />
```

Comment:
what is this?
Is it native intellij setting? Should we maintain this when/if we change the version of python supported in the templates?

Reply:
Good question. Let me test without it to see if it still works

Reply:
fixed. files removed